### DISCUSSION TITLE
Cannot connect via SSH using the default root username & password

### DISCUSSION THREAD

**Discussion by slavko98** [mgtcommerce/mgt-dev#3] [Q&A]
Hello all, I'm new to this project and also Docker, so please don't be too harsh :)

I was able to install the Docker instance of the MGT Development Environment, and I can connect to the panel under https://127.0.0.1:8443/domains

But when I try to connect via ssh with the default root/root username and password I get the "permission denied" error.

![docker ssh login error](https://user-images.githubusercontent.com/10711002/112452888-628cd500-8d57-11eb-9207-5242f4c246a2.png)

How can I check what's wrong, or perhaps can I try to somehow reset the settings to make sure I haven't changed something unwillingly?

**Comment by swieczorek** [comment]
Normally the root password is root.

You can also login via bash and reset it: docker exec -it mgt-dev bash 

Instead of "mgt-dev" you can also use the container id.

**Comment by p3ll3** [comment]
> Hello all, I'm new to this project and also Docker, so please don't be too harsh :)
> 
> I was able to install the Docker instance of the MGT Development Environment, and I can connect to the panel under https://127.0.0.1:8443/domains
> 
> But when I try to connect via ssh with the default root/root username and password I get the "permission denied" error.
> 
> ![docker ssh login error](https://user-images.githubusercontent.com/10711002/112452888-628cd500-8d57-11eb-9207-5242f4c246a2.png)
> 
> How can I check what's wrong, or perhaps can I try to somehow reset the settings to make sure I haven't changed something unwillingly?

You can try disabling the public key authentication like this:

`ssh -o PubkeyAuthentication=no root@127.0.0.1`

Hope it helps!
